scripts/get-gee-build-environment.sh

```diff
@@ -0,0 +1,432 @@
+#! /bin/bash
+
+# This scripts installs packages, downloads and buils software used to
+# download the Open GEE source code and build it.
+#   It runs on RHEL 6, RHEL 7, Cent OS 6, Cent OS 7,
+# Ubuntu 14.04, Ubuntu 16.04.
+#   This script does not share code with other shell scripts so that it can be
+# distributed as a single file that can be used to obtain a build environment
+# and source code for Open GEE.
+
+SELF_NAME=$(basename "$0")
+
+MIN_GCC_VERSION="4.8"
+
+# Where to build packages (RPMs) that are not provided with the distribution 
+# we're running on (when necessary):
+: ${PACKAGE_BUILD_DIR:=""}
+
+# Whether to delete the package build directory before exiting:
+: ${CLEAN_PACKAGE_BUILD_DIR:="yes"}
+
+# Where we download packages (RPMs) that are not provided in repositories
+# (Yum):
+: ${PACKAGE_DOWNLOAD_DIR:=""}
+
+# Whether to delete the package download directory before exiting:
+: ${CLEAN_PACKAGE_DOWNLOAD_DIR:="yes"}
+
+# Whether to assume answer of yes to all questions (used when installing
+# packages).  Set to true to run without user interaction.
+: ${ASSUME_YES:=""}
+
+# Open GEE repository URL for cloning:
```

Comment:
Reminder: Remove no longer used variables.

Reply:
Variable is still in use. Usage of check_root (mentioned below) was a mistake on my part

Reply:
I don't see where `CLONE_URL`, `CLONE_BRANCH` and `CLONE_REPOSITORY` are used.